site/en/blog/chrome-120-beta-whats-new-for-extensions/index.md

```diff
@@ -0,0 +1,113 @@
+
+As part of this work, we made some additional changes such as enforcing the limits on dynamic and session rulesets separately to align with Firefox. For more information, see our proposal.
+
+What makes me personally really happy about this change is how quickly it landed. [Oliver](https://twitter.com/oliverdunk_) wrote the initial proposal in August following his discussions with different ad blockers. It’s now landing in Chrome - two months later! 🎉
+
+## New ReadingList API
+
+Chrome introduced the reading list in 2021. Last year, the Chrome team made access to the reading list even easier via the side panel. With Chrome 120 we are adding the ability for Chrome Extensions to create, read, update, and delete reading list entries. To learn more, checkout the [API docs](/docs/extensions/reference/readingList/) and our [new sample](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/api-samples/readingList).
+
+<figure>
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/hbkhOAXaKLb6k3PI3zTp.png", alt="Chrome’s reading list showing extension doc pages.", width="754", height="1070" %}
+<figcaption>The reading list side panel in Chrome.</figcaption>
+</figure>
+
+
+
+ 
+## File handling on ChromeOS
+
+The File Handling API lets extensions open files with specified MIME types and file extensions in a similar manner to web platform file handling. Check out [File handling on ChromeOS](/docs/extensions/mv3/file_handling/) for more on how to use it.
+
+
+<figure>
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/DWpss6N4rnxu3BzfauRv.png", alt="Screenshot of the open file with extension dialog on ChromeOS", width="800", height="454" %}
+<figcaption>Open files in an extension on ChromeOS.</figcaption>
+</figure>
+
+
+## Trigger an alarm in 30 seconds
+
+This is a small update, but addresses an important gap in the service worker lifecycle. Due to the event driven nature of service workers, the recommended way to fire an event in the future is to use [`chrome.alarms`](/docs/extensions/reference/alarms/). The Alarms API ensures that the event gets fired even if the service worker shuts down in the meantime. 
+
+There’s a catch though. Before Chrome 120, the shortest timespan to trigger an alarm was 1 minute. However, service workers shut down after 30 seconds of inactivity. So there was no straightforward way to schedule an alarm to fire in 45 seconds, because when using [`setTimeout()`](https://developer.mozilla.org//docs/Web/API/setTimeout) to set an event in 45 seconds, the service worker could potentially be shut down before the event fired. 
```

Comment:
```suggestion
There’s a catch though. Before Chrome 120, the shortest time span to trigger an alarm was one minute. However, service workers shut down after 30 seconds of inactivity. So there was no straightforward way to schedule an alarm to fire in 45 seconds, because when using [`setTimeout()`](https://developer.mozilla.org//docs/Web/API/setTimeout) to set an event in 45 seconds, the service worker could potentially be shut down before the event fired. 
```

Reply:
>1 vs. one

I don't know what you do in Deutsch, but English does 1 through 9 different from larger numbers.